### PR TITLE
fix(gcf-utils): address octokit type regressions

### PR DESCRIPTION
**Pull request by bcoe**
~@octokit/rest` methods are now name-spaced under `github.rest`.~, I'm not sure what was forcing me to use namespaces, but this seems to have been corrected once I got all libraries updated appropriately.

### CODE REVIEW

**Comment by JustinBeckwith** [comment]
was the intent to delete the lock file?

**Comment by bcoe** [comment]
> was the intent to delete the lock file?

@JustinBeckwith no, I think it was removed because the postinstall script fails.

File: packages/gcf-utils/src/gcf-utils.ts [hunk 1]
```diff
@@ -387,16 +387,23 @@ export class GCFBootstrapper {
...
-        installationId,
-      },
-    });
+    const opts = {
+      appId: cfg.appId,
+      privateKey: cfg.privateKey,
+      installationId,
+    };
+    if (wrapOptions?.logging) {
+      const LoggingOctokit = Octokit.plugin(LoggingOctokitPlugin)
```
Comment (bcoe) [inline packages/gcf-utils/src/gcf-utils.ts:397]: The Probot Options types seemed to conflict with our approach of instantiating Octokit ourselves.

File: packages/gcf-utils/src/gcf-utils.ts [hunk 2]
```diff
@@ -109,7 +109,7 @@ export const addOrUpdateIssueComment = async (
   for (const comment of listCommentsResponse.data) {
     if (comment.body?.includes(commentMark)) {
       // We found the existing comment, so updating it
-      await github.issues.updateComment({
+      await github.rest.issues.updateComment({
```
Comment (bcoe) [inline packages/gcf-utils/src/gcf-utils.ts:112]: In the  newest version of `probot`, `github` is moved under the `rest` namespace.

File: packages/gcf-utils/src/gcf-utils.ts (hunk 1 above)
Comment (bcoe) [reply packages/gcf-utils/src/gcf-utils.ts:397]: we will want to test our cron based applications.

**Review by chingor13 — APPROVED** [review]

File: packages/gcf-utils/src/gcf-utils.ts (hunk 1 above)
Comment (tmatsuo) [reply packages/gcf-utils/src/gcf-utils.ts:397]: I tested it and failed🤣🤣

**Pull request merged**